lib/fileParser.js

```diff
@@ -268,13 +271,17 @@ function fileParser(filepath, options, addJobCallback) {
                     }
                     lastParsedFile = filepath[i];
                 }
+
                 return sortFile(tempFiles2, cleanupList);
             }).then(function (sortedFile) {
                 cleanupList.push(sortedFile);
                 return addJobsFromSortedFile(sortedFile, options, addJobCallback);
             }).then(function (result) {
                 core.log('info', 'Finished job creation... no more unemployment... at least for a bit');
                 return result;
+            }).catch(function() {
+                core.log('info', 'All tiles filtered out, nothing to process');
```

Comment:
I don't think catching and ignoring all errors is good. Its better to explicitly handle it by returning `false` or `null`, and handling it.